docs/core/howto/python3.rst

```diff
@@ -4,7 +4,7 @@ Porting to Python 3
 Introduction
 ------------
 
-Twisted currently supports only Python 3.6+.
+Twisted currently supports only Python 3.8+.
```

Comment:
Can we remove this line from here?

The minimum python version is documented inside the README and the pypi package metadata

Reply:
Ah, a middle ground.  `:]`  I don't really care.

```suggestion
```

Go ahead and commit this if you like the removal @cjwatson.

Reply:
Yes, I think removing this line makes sense.